openpype/hosts/maya/api/lib.py

```diff
@@ -1881,6 +1881,12 @@ def remove_other_uv_sets(mesh):
             cmds.removeMultiInstance(attr, b=True)
 
 
+def get_parent(node):
```

Comment:
I would rename it to `get_node_parent` for cleaner api.

I'm sometimes overthinking everything so you can resolve this without any action.